guides/source/form_helpers.md

```diff
@@ -290,95 +334,95 @@ so:
 <% end %>
 ```
 
-Which produces the following output:
+The above will produces the following output:
```

Comment:
_will produce_

Reply:
Hi @ruyrocha . Not sure if you are familiar with the suggestion button?
This would allow you add suggestion than can merged with a single click.
<img width="800" alt="image" src="https://github.com/rails/rails/assets/28561/5ff0b23a-f3ab-4b46-9f3f-43de0416ca6b">